torch/nn/_functions/bilinear.py

```diff
@@ -0,0 +1,66 @@
+import torch
+from torch.autograd import Function
+
+
+class Bilinear(Function):
+
+    def forward(self, input1, input2, weight, bias=None):
+        self.save_for_backward(input1, input2, weight, bias)
+
+        output = input1.new()
+
+        self.buff2 = input1.new()
```

Comment:
Removing self from buff2 isn't working as expected. We are using self.buff2 in backward that we initialized in forward. We can remove it from buff1 though. Do you want me to do it?